### DISCUSSION TITLE
the UI seems out-of-date

### DISCUSSION THREAD

**Discussion by CoderLeonidas** [ViennaRSS/vienna-rss#1585] [Ideas]
Vienna is good enough for me but its UI is really out-of-date. I hope for a new Vienna with a modern UI style such as "Clean my Mac"! Thanks!

**Comment by TAKeanice** [comment]
I agree that there are places in which the UI seems old-fashioned. A more specific list of issues or suggestions for design changes would be welcome!

**Comment by DavidMunch** [comment]
Relates to #88 and #764. I suggest updating the title here to "Suggestions for UI changes".

Personally I can come up with 6 suggestions: 
1. There's really only need for one search field (Covered in #764).
2. I'd like to see the tool bar buttons moved up next to the traffic lights, similar to how Safari does it. I have 2 buttons active, and the search field, in my toolbar, and on a 27" screen that's a LOOOOOT of wasted space. It would also make the app more lean, and thus look more fresh.
3. The Header when you selects a specific article currently has a dark grey background. I'd like to see it changed to the same dark blue color used for highlighting selected articles. It would bring some more color into the app.
4. Perhaps make the left pane background slightly transparent, as it seems to be all the rage these days.
5. The divider between the top and bottom panes for the 3 pane view, is really thick for no reason. The same divider between the left pane and the two on the right, is slim in contrast. I suggest making the top/bottom divider thin like the one on the left. UI consistency, and same functionality. As it is now, there's no indication that you can move it (Usually there's three dots or something), so I see no reason for it to be so thick.
6. Some kind of animation to indicate where new feeds are added (See #1163)

**Comment by neilt** [comment]
> Vienna is good enough for me but its UI is really out-of-date. I hope for a new Vienna with a modern UI style such as "Clean my Mac"! Thanks!

Please no.  Most of these modern UIs are all about removing information and making lots of white space in order to be fashionable, but not very usable.  We don't need changes because they are "all the rage these days."  Please no 2 or 3 inch icons with 2 and 3 inch white space around them.

Please do focus on usability, the lost art that left Apple about 8 to 10 years ago.

Lets continue to have buttons where one expects them, with pop up tips that eliminate the need to read the manual.  Lets NOT have to run the mouse around the screen to see if there are any hidden active areas (like Apple Mail), and then play wack-a-mole to find what you need.  Lets not have different menu options when you hold down the "option" key.  

You want an example of poor UI, I have to pull down a menu with the title "...." to edit this post.  What a poor example of usability.  But, at the same time, my screen shows 4 or 5 inches of unused white space around the post. The message header has 3 inches of white space with no buttons or anything.  It is fashionable, but has a low usability rating.

**Comment by TAKeanice** [comment]
> > Vienna is good enough for me but its UI is really out-of-date. I hope for a new Vienna with a modern UI style such as "Clean my Mac"! Thanks!
> 
> Please no. Most of these modern UIs are all about removing information and making lots of white space in order to be fashionable, but not very usable. We don't need changes because they are "all the rage these days." Please no 2 or 3 inch icons with 2 and 3 inch white space around them.
> 

I agree that hidden menus and wasted space are not the best idea (not the "yellow of the egg", as we say in Germany :D ). However, we could work on clarity, strip skeumorphisms and adapt style elements from the current OS. We did so with the Tabbar. We can as well overhaul the browser interface. I think everything else would be to adopt new gestures used across the OS and stylize and color the existing elements. Such a change was done to the buttons in the top bar for example.

**Comment by neilt** [comment]
Improvement in usability, consistency, and clarity are always welcome.

**Comment by barijaona** [comment]
I guess https://github.com/ViennaRSS/vienna-rss/discussions/1798 is proof that we have done our homework ? 😄
